gatling-recorder/src/main/scala/io/gatling/recorder/scenario/template/RequestTemplate.scala

```diff
@@ -107,10 +107,15 @@ private[scenario] object RequestTemplate {
       else
         ""
     val prefix = if (config.http.useSimulationAsPrefix) simulationClass else "request"
-    s"""http("${prefix}_${request.id}")
+    val postfix = if (config.http.useMethodAndUriAsPostfix) s":${sanitizeRequestPostfix(s"${request.method}_${request.uri}")}" else ""
```

Comment:
`s":${sanitizeRequestPostfix(s"${request.method}_${request.uri}")}"` => `sanitizeRequestPostfix(s"${request.method}_${request.uri}"`

Reply:
You are right, this was too complicated. As I need `:` as prefix, I changed it to:

    ":" + sanitizeRequestPostfix(s"${request.method}_${request.uri}")